src/main/java/com/redhat/rhjmc/containerjfr/platform/PlatformClient.java

```diff
@@ -43,6 +43,8 @@
 
 import java.util.List;
 
+import com.redhat.rhjmc.containerjfr.core.log.Logger;
+
 public interface PlatformClient {
-    List<ServiceRef> listDiscoverableServices();
+    List<ServiceRef> listDiscoverableServices(Logger logger);
```

Comment:
This seems wrong. Why does the `Logger` need to be provided as an argument to this method? I would expect it to be dependency injected into the `PlatformClient` implementations' constructors.

Reply:
I removed it!